README.md

```diff
@@ -57,150 +65,132 @@ Note: In this example configuration above required &quot;ipam&quot; is provided
+Configuration is based on the Multus CRD Network Attachment Definition.  Please follow the configuration details in the link: [Usage with Kubernetes CRD based Network Objects](https://github.com/intel/multus-cni/blob/master/doc/configuration.md#configuration-example)
 
-Please refer to the [K8s Multiple Network PoC proposal](https://docs.google.com/document/d/1TW3P4c8auWwYy-w_5afIPDcGNLK3LZf0m14943eVfVg/edit)for more details.
+For more information and advanced use refer to the [Network Custom Resource standard](https://docs.google.com/document/d/1TW3P4c8auWwYy-w_5afIPDcGNLK3LZf0m14943eVfVg/edithttps://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit#heading=h.hylsbqoj5fxd) for more details.
 
-### Configuration details
-```json
-# cat > /etc/cni/net.d/00-multus.conf <<EOF
-{                                                                        
-    "name": "multus-demo-network",                                       
-    "type": "multus",                                                    
-    "delegates": [                                                       
-        {
-            "type": "sriov",
-            "if0": "ens6f0",
-            "if0name": "net0",
-            "l2enable": true
-        },
-        {
-            "type": "sriov",
-            "if0": "ens6f3",
-            "if0name": "net1",
-            "l2enable": true
-        },
-        {
-            "type": "bond",
-            "ifname": "bond0",
-            "mode": "active-backup",
-            "miimon": "100",
-            "links": [
-                    {"name": "net0"},
-                    {"name": "net1"}
-            ],
-
-            "ipam": {
-                 "type": "host-local",
-                 "subnet": "192.168.1.0/24",
-                 "rangeStart": "192.168.1.21",
-                 "rangeEnd": "192.168.1.30",
-                 "routes": [
-                      { "dst": "0.0.0.0/0" }
-                 ],
-                 "gateway": "192.168.1.1"
-            }
-        },
-        {
-            "type": "flannel",
-            "name": "control-network",
-            "masterplugin": true,
-            "delegate": {
-                    "isDefaultGateway": true
-            }
-        }
-    ]
-}
-EOF
-```
 #### Launching workloads in Kubernetes
 
-With above Multus configuration, we can now deploy a Pod using the pod spec shown below. (Assuming bond, sriov and flannel CNI  binaries are present in default CNI location). Once successfully created, this Pod should have multiple network interfaces attached to it.
+Prerequisites:
+
+- Multus configured as per the [quick start guide](https://github.com/intel/multus-cni/blob/master/doc/quickstart.md)
+
+- SRIOV CNI and Multus CNI placed in /opt/cni/bin
+
+1) Deploy Network Attach Definiton for SRIOV:
+
+```
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
Shall we change the SR-IOV example NAD to not contain IPAM config? With bond inside container, it seems not necessary.

Reply:
IPAM removed